src/main/riddl/Store/Store.riddl

```diff
@@ -3,7 +3,6 @@ context StoreContext is {
   type StoreId is Id(StoreContext.Store)
 
   type StoreInfo is {
-    id: StoreId,
```

Comment:
I am assuming that you are pulling storeId from StoreInfo and calling out storeId as an attribute in each of the state definitions? But I don't see that change. So where does StoreId go?